docs/tokens/lockup.md

```diff
@@ -0,0 +1,118 @@
+
+Next, the account [9b84742f269952cea2877425b5e9d2e15cae8829.lockup.near](https://explorer.mainnet.near.org/accounts/9b84742f269952cea2877425b5e9d2e15cae8829.lockup.near) was created to store the actual balance of locked tokens on the account in [a batch transaction](https://explorer.mainnet.near.org/transactions/Eer14Fih17TRjpiF8PwWfVKNTB57vXnNJsDW93iqc2Ui) which also transferred these tokens to it (in this case, 594.11765 tokens).  You can see the arguments for the `new` method in the explorer, which show a 12-month release duration with an initial cliff of October 4th:
+
+```
+{
+  "owner_account_id": "gio3gio.near",        // the Owner account who is allowed to call methods on this one
+  "lockup_duration": "0",                    // not necessary if the lockup_timestamp is used
+  "lockup_timestamp": "1601769600000000000", // Unix timestamp for October 4th, 2020 at midnight UTC
+  "transfers_information": {
+    "TransfersDisabled": {
+      "transfer_poll_account_id": "transfer-vote.near"
+    }
+  },
+  "vesting_schedule": null,
+  "release_duration": "31536000000000000",  // 365 days
+  "staking_pool_whitelist_account_id": "lockup-whitelist.near",
+  "foundation_account_id": null
+}
+```
+
+For the actual lockup contract code and README, [see it on Github](https://github.com/near/core-contracts/tree/master/lockup).
+
+
+## Delegating locked tokens
+
+One of the unique features of the NEAR lockups is ability to delegate tokens while they are still locked.
+
+There are few things to know:
+1. You can only delegate to whitelisted pools, right now it's all the pools that end with `.poolv1.near`.
+2. One lockup contract can only delegate to a single pool.
+3. The account must keep minimum balance of 35 $NEAR to cover storage for lockup contract itself (transactions that will try withdraw over that amount will just fail).
+4. Delegation rewards can be withdrawn back to the lockup contract but are unlocked, so they can be withdrawn from it right away.
+5. Delegating commands / tools which are not specifically configured to work with locked-up accounts won't work, as the "owner account" must call lockup contract. Currently Dokia and NEAR Wallet are adding native support for lockup contract delegation.
+
+
+## Frequent questions?
+
+### I don't see my full balance in my wallet
+
+Not all wallets support looking up the locked-up balance.
+
+There are three ways to go:
+ - Check your balance using https://near.github.io/account-lookup/.
+ - [Import your account into NEAR Wallet](token-custody#importing-accounts-from-other-wallets).
+ - Use CLI to check your balance: `near view <LOCKUP_ACCOUNT_ID> get_balance ''` (note it outputs the value in yoctoNEAR - divide by 10e24 to get NEAR amount).
+
+### How do I delegate locked up tokens?
+
+It's important to note, that a single lockup contract can only delegate to a single validator.
+
+Go to Dokia's staking UI: https://staking.dokia.cloud/staking/near/validators
```

Comment:
we should also link to instructions on how to do this through near-cli. Also I am not sure whether we want to endorse dokia's portal officially.